account_banking_mandate_sale/oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+sale-workflow
```

Comment:
This file `oca_dependencies.txt` should be not placed inside the module. Use the one already existing in the repo https://github.com/OCA/bank-payment/tree/16.0

Reply:
You're right, I don't know who put it there it comes from a previous version. I'll remove it